Renderer.js

```diff
@@ -0,0 +1,14 @@
+const Renderer = function (data) {
+
+	const render = function () {
+		$("#container-for-template").empty()
+		const soruce = $("#tweeter-template").html()
+		const template = Handlebars.compile(soruce)
+		
+			const someHTML = template({posts: data, comments:data.comments })
```

Comment:
You shouldn't send the `posts` and `comments` separately since `comments` should be included in each `post`.